indexer-watcher/src/Notification.ts

```diff
@@ -93,7 +93,7 @@ export class DailyReport implements Notification {
     public readonly level = "info";
     public readonly date = new Date();
 
-    public constructor(firstBlockNumber: number, lastBlockNumber: number) {
+    public constructor(firstBlockNumber: number | string, lastBlockNumber: number) {
```

Comment:
How about `firstBlockNumber: number | "unknown"`?

Reply:
I changed the type to use the "unknown".